### ISSUE TITLE
Replace commends

### ISSUE THREAD

**Issue by Redo02** [ValveSoftware/Dota2-Gameplay#13344]
### Description

Since commend wont increase your behavior. what is point of commend???
recieves Commends higher than report but we still lose score.

so they must remove commends. replace with report counter. only thing they care about. is report
lower behavior never able to increase score now.

**Comment by Aluerie** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11278

**Comment by BledNavalny** [comment]
Yeah, it was more sense before 2016 when it show how many commend you got;)
